admin/bin/qserv-admin.py

```diff
@@ -636,14 +637,14 @@ def main():
             try:
                 parser.parse(cmd)
             except (_ToolError, css.CssError) as e:
-                print "ERROR: ", e.__str__()
+                print("ERROR: ", e.__str__())
```

Comment:
Not related to modernization, but should not use special methods like `__str__` directly, we'll need to make another round of cleanup for those I guess.